### ISSUE TITLE
Perfil: Location not available

### ISSUE THREAD

**Issue by numapanuma** [Semillas/SemillasReactNative#3]
No hay campo para poner la dirección/ubicación. Será porque tengo el GPS desactivado?

**Comment by iesteban** [comment]
Sí, en un principio no hay forma para el usuario de meter la localización. 

En cualquier caso estoy barajando si la localización debería ser por cada servicio que se ofrece, o bien la localización del usuario.

**Comment by numapanuma** [comment]
Ah ok. Tal vez sería bueno poner la locación del usuario por defecto para que sea más simple, pero con la posibilidad de cambiar a "global" en el caso de servicios que se puedan brindar virtualmente..
